x/validator-preference/keeper_test.go

```diff
@@ -0,0 +1,67 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/osmosis-labs/osmosis/v12/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v12/x/validator-preference/types"
+	"github.com/stretchr/testify/suite"
+)
+
+type KeeperTestSuite struct {
+	apptesting.KeeperTestHelper
+
+	cleanup func()
+}
+
+func (suite *KeeperTestSuite) SetupTest() {
+	suite.Setup()
+}
+
+func (suite *KeeperTestSuite) Cleanup() {
+	suite.cleanup()
+}
+
+func (suite *KeeperTestSuite) SetupValidators(bondStatuses []stakingtypes.BondStatus) []sdk.ValAddress {
+	valAddrs := []sdk.ValAddress{}
+	for _, status := range bondStatuses {
+		valAddr := suite.SetupValidator(status)
+		valAddrs = append(valAddrs, valAddr)
+	}
+	return valAddrs
+}
```

Comment:
nit: If the other function sets up multiple validators, why not just have this function take a single staking type and return a single val address? Would just make its use in SetupMultipleValidators less awkward imo. Feel free to ignore though!

Reply:
Yup this approach is very clean. I'll update it